release.sh

```diff
@@ -25,28 +29,54 @@ changelog() {
 }
 
 bump() {
-  # Bump package version and generate changelog
-  VERSION="${NEXT/v/}"
 
-  echo "Bump version to ${VERSION}"
+  if [ "$TRAVIS_PULL_REQUEST" != "false" ]; then
+      printfln "Exiting for pull request without versioning."
```

Comment:
This doesn't exit the script. I'm confused. Should we exit here? It only prints.

Reply:
Oh will fix it